src/aks/subscriptions/PROD-CSTAR/terraform.tfvars

```diff
@@ -7,6 +7,8 @@ ingress_load_balancer_ip = "10.1.0.250"
 
 rbac_namespaces = ["bpd", "rtd", "fa"]
 
+enable_postgres_replica = true
```

Comment:
we decided to not connect to replica anymore ..... even in production.

Reply:
but it's a variable enabled only in production.
If we need to do a performance test in uat we have a variable to switch the connection...

Reply:
thanks, microservices use only master postgres.